skill-manager/api_server.http

```diff
@@ -0,0 +1,136 @@
+@hostname = square.ukp-lab.de
```

Comment:
is this file really required? If the only thing changes is the hostname it would be better to just edit it in the file to the host that you are targeting. When having two copies of the same file they will get out of sync for sure 😅

Reply:
strictly speaking, it's not necessary, but it is very beneficial for future developers. I added a lot of possible api calls to the skill manager or related. The change is not the hostname, but all other api calls.

Reply:
Ok, i might need to have a closer look. But why not just extend api.http?

Reply:
oh! you mean that! sorry you are right. I think this file was made as a backup and I guess accidentally I included in the commit

Reply:
I have deleted it